games-fps/prboom-plus/metadata.xml

```diff
@@ -12,14 +12,6 @@
 		variable gamespeed, re-record, walkcam, chasecam, full mouselook, FOV,
 		and other features without loss of compatibility with the original Doom.
 	</longdescription>
-	<maintainer type="person" proxied="yes">
```

Comment:
Really, I think if you feel so strongly about it being unmaintainable in Gentoo because upstream is gone, we should last-rite it. But not having an upstream isn't a reason to last-rite it if it works otherwise _and_ someone wants to keep it.

Reply:
I was planning to just "move" it to the new fork.

Reply:
That's an interesting idea. I think the original team members are maintaining the new fork so that could work. How would you migrate Gentoo users to the new fork package?

Reply:
You add an entry to profiles/updates in the repo.